requirements.txt

```diff
@@ -3,4 +3,5 @@ py-cpuinfo
 pydataset
 system_query[cpu, hdd, ram, swap]
 python_papi
-# install system_query[all] for GPU systems
\ No newline at end of file
+tensorflow-gpu
```

Comment:
I don't think there's GPU on travis, so can install tensorflow without gpu
also let's move frameworks to test_requirements